docs/configuration/settings.rst

```diff
@@ -480,6 +480,28 @@ In order to replace it by another one:
     multiauth.policy.basicauth.use = myproject.authn.BasicAuthPolicy
 
 
+Mohawk request Auth
+:::::::::::::::::::
+
+`Mohawk <https://mohawk.readthedocs.io/en/latest/index.html>`_ implements the 
+`Hawk authorization scheme <https://github.com/hueniverse/hawk>`_, which 
+allows you to authenticate messages from your application backend to Kinto.
+In order to prevent replay attacks, nonce checking is enforced.  
+See `here <https://github.com/hueniverse/hawk#implementations>` for 
+some implementations of Hawk in different programming languages.
+
+Mohawk auth is enabled by turning on ``kinto.hawk.enabled`` and providing 
+credential values for ``kinto.hawk.id``, ``kinto.hawk.secret``, and
+``kinto.hawk.algo``.  
+
+.. code-block:: ini
+
+    kinto.hawk.enabled = true
+    kinto.hawk.id = 'some client id'
+    kinto.hawk.secret = 'some secret key'
+    kinto.hawk.algo = 'sha256'
+
```

Comment:
Also the `account` plugin must be enabled as far as I understand

Reply:
thanks for reading through my docs change!  Most of the change is outdated and I shouldn't have included it in the PR.  Since we are moving in the direction of basically using hawk auth as a wrapper around the account plugin, I will need to redo this.  Your comments are helpful in understanding what needs to be in the docs